### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update several items in the README
| Type?         | improvement
| Category?     | PM
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No need

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15518)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ The 'develop' branch of this repository contains the work in progress source cod
...
 For more information on our branch system, read our guide on [installing PrestaShop for development][install-guide-dev].
 
-PRESTASHOP 1.7 IS NOW PRODUCTION-READY! Its first stable version, 1.7.0.0, was released on November 7th, 2016. Further updates have been released since then. Learn more about it on [the Build devblog](http://build.prestashop.com/tag/1.7/).
+The first stable version of PrestaShop 1.7, 1.7.0.0, was released on November 7th, 2016. Further updates have been released since then. Learn more about it on [the Build devblog](https://build.prestashop.com/tag/1.7/).
 
 Server configuration
 --------
 
-To install PrestaShop 1.7, you need a web server running PHP 5.6+ and any flavor of MySQL 5.0+ (MySQL, MariaDB, Percona Server, etc.).
+To install PrestaShop 1.7, you need a web server running PHP 5.6+ and any flavor of MySQL 5.0+ (MySQL, MariaDB, Percona Server, etc.). Latest versions of PrestaShop 1.7 require PHP 7.1+.
```
Comment (PierreRambaud) [inline README.md]:
```suggestion
To install the latest PrestaShop 1.7, you need a web server running PHP 7.1+ and any flavor of MySQL 5.0+ (MySQL, MariaDB, Percona Server, etc.). Versions between 1.7.0 and 1.7.6 work with PHP 5.6+.
```

File: README.md [hunk 2]
```diff
@@ -21,12 +21,12 @@ The 'develop' branch of this repository contains the work in progress source cod
...
 For more information on our branch system, read our guide on [installing PrestaShop for development][install-guide-dev].
 
-PRESTASHOP 1.7 IS NOW PRODUCTION-READY! Its first stable version, 1.7.0.0, was released on November 7th, 2016. Further updates have been released since then. Learn more about it on [the Build devblog](http://build.prestashop.com/tag/1.7/).
+The first stable version of PrestaShop 1.7, 1.7.0.0, was released on November 7th, 2016. Further updates have been released since then. Learn more about it on [the Build devblog](https://build.prestashop.com/tag/1.7/).
 
 Server configuration
 --------
 
-To install PrestaShop 1.7, you need a web server running PHP 5.6+ and any flavor of MySQL 5.0+ (MySQL, MariaDB, Percona Server, etc.).
+To install the latest PrestaShop 1.7, you need a web server running PHP 7.1+ and any flavor of MySQL 5.0+ (MySQL, MariaDB, Percona Server, etc.). Versions between 1.7.0 and 1.7.6 work with PHP 5.6+.
```
Comment (Progi1984) [inline README.md:29]: @PierreRambaud Always in MySQL 5.0 ?

Comment (PierreRambaud) [reply README.md:29]: Right now, yes. The code needs to be updated to change the minimum MySQL version :/

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for reviews !

**Pull request merged**